WordPress/Classes/ViewRelated/Comments/Controllers/CommentDetailViewController.swift

```diff
@@ -446,10 +446,16 @@ private extension CommentDetailViewController {
             self?.openWebView(for: url)
         }
 
-        cell.accessoryButtonType = .info
-        cell.isAccessoryButtonEnabled = true
-        cell.accessoryButtonAction = { [weak self] senderView in
-            self?.presentUserInfoSheet(senderView)
+        if comment.allowsModeration() {
+            cell.accessoryButtonType = .info
+            cell.isAccessoryButtonEnabled = true
+            cell.accessoryButtonAction = { [weak self] senderView in
+                self?.presentUserInfoSheet(senderView)
+            }
+        } else {
```

Comment:
Update `accessoryButtonType` and `isAccessoryButtonEnabled` here to match the `if true` branch, to avoid potential cell reusing issues.